packages/aragon-wrapper/src/utils/index.js

```diff
@@ -21,7 +21,12 @@ export function includesAddress (arr, address) {
 
 // Address map that ensures consistent non-checksummed interpretations of addresses
 export function makeAddressMapProxy (target = {}) {
-  return new Proxy(target, {
+  const targetLowerCaseKeys = {}
+  Object.keys(target).forEach(addr => {
+    targetLowerCaseKeys[addr.toLowerCase()] = target[addr]
+  })
```

Comment:
Super small nitpick: we could use `Object.entries` to directly get the target value for each key